contributor_docs/hi/contributor_guidelines.md

```diff
@@ -0,0 +1,530 @@
+
+`[your_commit_message]` को एक प्रासंगिक प्रतिबद्ध संदेश के साथ प्रतिस्थापित किया जाना चाहिए जो सामान्य बयानों से बचते हुए परिवर्तनों का वर्णनात्मक हो। उदाहरण के लिए, 'डॉक्यूमेंटेशन फिक्स 1' कहने के बजाय, 'सर्कल() फ़ंक्शन में दस्तावेज़ उदाहरण जोड़ें' कहें।
+
+```
+git commit -m "Add documentation example to circle() function"
+```
+
+यह सुनिश्चित करने के लिए कि चीजें काम कर रही हैं, समय-समय पर `npm test` चलाना सुनिश्चित करते हुए आपके द्वारा किए जा रहे सभी कमिट के लिए उपरोक्त चरणों को दोहराएं।
+
+### सोर्स कोड
+
+यदि आप स्रोत कोड पर काम करने जा रहे हैं, तो शुरुआत करने के लिए एक अच्छी जगह है, यदि आप जानते हैं कि आप किस p5.js सुविधाओं पर काम करने जा रहे हैं, तो प्रलेखन पर जाएँ और p5.js की प्रत्येक प्रलेखित कार्यक्षमता के नीचे जाएँ। इसके सोर्स कोड का लिंक होगा।
+
+![p5.js वेबसाइट पर एक संदर्भ पृष्ठ का क्रॉप किया गया स्क्रीनशॉट जिसमें वाक्य है "किसी भी त्रुटि या टाइपो पर ध्यान दें? कृपया हमें बताएं। कृपया बेझिझक src/core/shape/2d\_primitives.js को संपादित करें और एक पुल अनुरोध जारी करें !". उपरोक्त वाक्य का वह भाग जहां यह कहता है "src/core/shape/2d\_primitives.js" को लाल रेखा और तीर की ओर इंगित करते हुए हाइलाइट किया गया है।](../images/reference-code-link.png)
+
+### इकाई का परीक्षण
+
+यदि आप इकाई परीक्षण पर काम करने जा रहे हैं, तो कृपया [यहां](https://github.com/processing/p5.js/blob/main/contributor_docs/unit_testing.md) देखें। ध्यान दें कि किसी भी सुविधा वृद्धि, नई सुविधाओं और कुछ त्रुटि फिक्स के लिए, नए कार्यान्वयन को कवर करने वाले यइकाई परीक्षणों को पीआर में शामिल किया जाना चाहिए।
+
+### इनलाइन दस्तावेज़ीकरण
+
+यदि आप इनलाइन दस्तावेज़ीकरण पर काम करने जा रहे हैं, तो कृपया [यहां](https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md) देखें।
+
+### अभिगम्यता
+
+यदि आप अभिगम्यता सुविधाएँ पर काम करने जा रहे हैं, तो कृपया [यहां](https://github.com/processing/p5.js/blob/main/contributor_docs/web_accessibility.md) देखें। एक अनुकूल त्रुटि प्रणाली के लिए, कृपया [यहां](https://github.com/processing/p5.js/blob/main/contributor_docs/friendly_error_system.md) देखें।
+
+## कोड मानक
+
+p5.js का कोड मानक या कोड शैली [ESLlint](https://eslint.org/) द्वारा लागू की जाती है। किसी भी गिट कमिट और पुल अनुरोध को स्वीकार करने से पहले उसे लिंटिंग से गुजरना होगा। आपके लिए सही कोडिंग मानक का पालन करने का सबसे आसान तरीका आपके टेक्स्ट एडिटर के लिए लिंटिंग एरर हाइलाइटिंग (अधिकांश लोकप्रिय टेक्स्ट संपादकों के लिए उपलब्ध) के साथ उपलब्ध ESLint प्लगइन का उपयोग करना है।
+
+## सॉफ़्टवेयर डिज़ाइन सिद्धांत
+
+p5.js की किसी भी विशेषता पर काम करते समय, p5.js के डिज़ाइन सिद्धांतों को ध्यान में रखना महत्वपूर्ण है। हमारी प्राथमिकताएँ अन्य परियोजनाओं की प्राथमिकताओं से भिन्न हो सकती हैं, इसलिए यदि आप किसी भिन्न परियोजना से आ रहे हैं, तो हम अनुशंसा करते हैं कि आप स्वयं को p5.js के डिज़ाइन सिद्धांतों से परिचित करा लें।
+
+- **पहुंच** हम सबसे पहले पहुंच को प्राथमिकता देते हैं, और हमारे द्वारा लिए गए निर्णयों में इस बात को ध्यान में रखना चाहिए कि यह ऐतिहासिक रूप से हाशिए पर रहने वाले समूहों तक पहुंच कैसे बढ़ाता है। हमारे पहुंच विवरण में इसके बारे में और पढ़ें।
```

Comment:
"उपलब्धता" will be better for accessibility

Reply:
Done

Reply:
done